Character/Classes/Sorcerer/Sorcerer.xml

```diff
@@ -226,7 +228,7 @@
 			</feature>
 			<feature optional="YES">
 				<name>Metamagic: Twinned Spell</name>
-				<text>When you cast a spell that targets only one creature and doesn't have a range of self, you can spend a number of sorcery points equal to the spell's level to target a second creature in range with the same spell (1 sorcery point if the spell is a cantrip).</text>
+				<text>When you cast a spell that doesn't have a range of self and is incapable of targeting more than one creature at the spell's current level, you can spend a number of sorcery points equal to the spell's level to target a second creature in range with the same spell (1 sorcery point if the spell is a cantrip).</text>
```

Comment:
I am not sure where this wording came from, the previous is consistent with the PHB